buf/registry/module/v1/label_service.proto

```diff
@@ -54,6 +54,12 @@ service LabelService {
   rpc ArchiveLabels(ArchiveLabelsRequest) returns (ArchiveLabelsResponse) {
     option idempotency_level = IDEMPOTENT;
   }
+  // Unarchive existing Labels.
+  //
+  // This operation is atomic. Either all Labels are unarchived or an error is returned.
```

Comment:
Question that's also relevant for `ArchiveLabels` RPC: what if they're trying to unarchive something that's already unarchived? I'd say that's ok and we unarchive all the rest?

Reply:
Yes, that makes sense to me and is how I would implement this RPC.